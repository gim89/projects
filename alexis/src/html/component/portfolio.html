<section class="portfolio section__padding section__text">
  <div class="container">
    <h2 class="portfolio__title">Portfolio.</h2>
    <p class="portfolio__subtitle subtitle__margin">
      We understand your requirement and provide quality works.
    </p>
    <div class="portfolio__body section__body">
      <img class="portfolio__item" src="img/portf-one.jpg" alt="portf-img-one">
      <img class="portfolio__item" src="img/portf-two.jpg" alt="portf-img-two">
      <img class="portfolio__item" src="img/portf-three.jpg" alt="portf-img-three">
      <img class="portfolio__item" src="img/portf-four.jpg" alt="portf-img-four">
      <img class="portfolio__item" src="img/portf-five.jpg" alt="portf-img-five">
      <img class="portfolio__item" src="img/portf-six.jpg" alt="portf-img-six">
      <img class="portfolio__item" src="img/portf-seven.jpg" alt="portf-img-seven">
      <img class="portfolio__item" src="img/portf-eight.jpg" alt="portf-img-eight">
      <img class="portfolio__item" src="img/portf-nine.jpg" alt="portf-img-nine">
    </div>
  </div>
</section>