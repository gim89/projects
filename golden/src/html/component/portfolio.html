<section id="portfolio" class="portfolio section__padding">
  <div class="container">
    <h2 class="portfolio__title">our portfolio</h2>
    <p class="portfolio__subtitle subtitle__margin">Lorem ipsum dolor sit amet consectetur.</p>
    <div class="portfolio__body">
      <div class="portfolio__item">
        <img src="img/portfolio-one.jpg" alt="" class="portfolio__img">
        <h3 class="portfolio__caption">Ebony Ivory</h3>
        <p class="portfolio__text">Branding</p>
      </div>
      <div class="portfolio__item">
        <img src="img/portfolio-two.jpg" alt="" class="portfolio__img">
        <h3 class="portfolio__caption">Smart Stationary</h3>
        <p class="portfolio__text">Print Design</p>
      </div>
      <div class="portfolio__item">
        <img src="img/portfolio-three.jpg" alt="" class="portfolio__img">
        <h3 class="portfolio__caption">Clever Poster</h3>
        <p class="portfolio__text">Print Design</p>
      </div>
      <div class="portfolio__item">
        <img src="img/portfolio-four.jpg" alt="" class="portfolio__img">
        <h3 class="portfolio__caption">Vinyl Record</h3>
        <p class="portfolio__text">Product Mock-Up</p>
      </div>
      <div class="portfolio__item">
        <img src="img/portfolio-five.jpg" alt="" class="portfolio__img">
        <h3 class="portfolio__caption">Treehouse Template</h3>
        <p class="portfolio__text">Web Design</p>
      </div>
      <div class="portfolio__item">
        <img src="img/portfolio-six.jpg" alt="" class="portfolio__img">
        <h3 class="portfolio__caption">Burned Logo</h3>
        <p class="portfolio__text">Branding</p>
      </div>
    </div>
  </div>
</section>